docs/modules/Record.ts.md

```diff
@@ -101,16 +104,33 @@ Added in v2.0.0
 
 ## compact
 
+Compact a `Record` of options removing the `none` values and
+keeping the `some` values.
```

Comment:
I think it's clearer and more consistent to refer to the types rather than their constructors, in which case `none` and `some` should be referred to as `None` and `Some`.

Reply:
👌